Example/Tests/ViewModels/AuthorizationsViewModelSpec.swift

```diff
@@ -41,7 +41,7 @@ final class AuthorizationsViewModelSpec: BaseSpec {
         describe("emptyViewData") {
             context("when there is no connections") {
                 it("should return correct data") {
-                    let viewModel = AuthorizationsViewModel()
+                    let viewModel = ConnectionsViewModelSpecAuthorizationsViewModel()
```

Comment:
what is this? We don't have this in code

Reply:
How did you run specs without failure?

Reply:
it was an intermediate commit, in which all the tests were not run at once.